.gitignore

```diff
@@ -1 +0,0 @@
-/.vscode
```

Comment:
Did you mean to remove this from the .gitignore?

Reply:
No. I hope that's been cleared up now (Brent helped me with it).